great_expectations/core/usage_statistics/anonymizers/profiler_run_anonymizer.py

```diff
@@ -0,0 +1,196 @@
+        """
+
+        name: Optional[str] = kwargs.get("name")
+        anonymized_name: Optional[str] = self.anonymize(name)
+
+        config_version: Union[float, str] = kwargs.get("config_version", 1.0)
+        variable_count: int = kwargs.get("variable_count", 0)
+        rule_count: int = kwargs.get("rule_count", 0)
+
+        rules: Dict[str, dict] = kwargs.get("rules", {})
+        anonymized_rules: List[dict] = self._anonymize_rules(rules)
+
+        anonymized_profiler_run_properties_dict: dict = {
+            "anonymized_name": anonymized_name,
+            "config_version": config_version,
+            "variable_count": variable_count,
+            "rule_count": rule_count,
+            "anonymized_rules": anonymized_rules,
+        }
+
+        deep_filter_properties_iterable(
+            properties=anonymized_profiler_run_properties_dict,
+            clean_falsy=True,
+            inplace=True,
+        )
+
+        return anonymized_profiler_run_properties_dict
+
+    def _anonymize_rules(self, rules: Dict[str, dict]) -> List[dict]:
+        anonymized_rules: List[dict] = []
+
+        for name, rule in rules.items():
+            anonymized_rule: dict = self._anonymize_rule(name, rule)
+            anonymized_rules.append(anonymized_rule)
+            logger.debug("Anonymized rule %s", name)
+
+        return anonymized_rules
+
+    def _anonymize_rule(self, name: str, rule: dict) -> dict:
+        anonymized_rule: dict = {}
+        anonymized_rule["anonymized_name"] = self.anonymize(name)
+
+        domain_builder: Optional[dict] = rule.get("domain_builder")
+        if domain_builder is not None:
+            anonymized_rule[
+                "anonymized_domain_builder"
+            ] = self._anonymize_domain_builder(domain_builder)
+
+        parameter_builders: List[dict] = rule.get("parameter_builders", [])
+        anonymized_rule[
+            "anonymized_parameter_builders"
+        ] = self._anonymize_parameter_builders(parameter_builders)
+
+        expectation_configuration_builders: List[dict] = rule.get(
+            "expectation_configuration_builders", []
+        )
+        anonymized_rule[
+            "anonymized_expectation_configuration_builders"
+        ] = self._anonymize_expectation_configuration_builders(
+            expectation_configuration_builders
+        )
+
+        return anonymized_rule
+
+    def _anonymize_domain_builder(self, domain_builder: dict) -> dict:
+        anonymized_domain_builder: dict = {}
+        anonymized_domain_builder["class_name"] = domain_builder.get("class_name")
+
+        batch_request: Optional[dict] = domain_builder.get("batch_request")
+        if batch_request:
+            anonymized_domain_builder[
+                "anonymized_batch_request"
+            ] = self._batch_request_anonymizer.anonymize_batch_request(
+                *(), **batch_request
```

Comment:
Just for my understanding, is the `*()` necessary because the method signature has `args*` e.g.  `anonymize_batch_request(*args, **kwargs)`?

Reply:
This was just a pattern I copied over from the checkpoint run anonymizer without thinking 😅 

The `*()` is not necessary here; I've gone ahead and removed it.